README.md

```diff
@@ -1 +1,91 @@
-repository for dkjson from http://dkolf.de/src/dkjson-lua.fsl
+This is the LUA dkjson module from [http://dkolf.de/dkjson-lua](http://dkolf.de/dkjson-lua)
```

Comment:
Please use `Lua` with a capital L as below
Please note that this file was intended to be empty and I may revert it on next update